docs/source/tools/text_normalization.rst

```diff
@@ -0,0 +1,34 @@
+Text Normalization
+==================
+
+Text normalization system for english, e.g. `123 kg` -> `one hundred twenty three kilograms`
```

Comment:
More examples and list of capabilities here? does it do de-normalization too?

Reply:
no, only normalization, up to @yzhang123 to add more in a separate PR